docs/Tasks/installing-an-extension.md

```diff
@@ -1,62 +1,116 @@
+
+     `version`
+     : Optional: Specifies the version or version range you want installed, such as `1.3.1` or `"<2"`.
+     If you use a comparison string to define a version range, the string must be surrounded by double quotes (`"`).
+
+    !!! warning
+        Currently, the following limitations affect the installation of extensions:
+
+        * If mulitple catalogs are added to a cluster, you cannot specify a catalog when you install an extension.
+        * OLM 1.0 requires that all of the extensions have unique bundle and package names for dependency resolution.
+
+        As a result, if two catalogs have an extension with the same name, the installation might fail or lead to an unintended outcome.
+
+2. Apply the CR the cluster:
+
+    ``` terminal
+    $ kubectl apply -f <cr_name>.yaml
+    ```
+
+    ??? success
+        ``` text title="Example output"
+        clusterextension.olm.operatorframework.io/argocd-operator created
+        ```
+
+### Verification
+
+* Get information about your bundle deployment:
+
+    ``` terminal
+    $ kubectl get bundledeployment
+    ```
+
+    ??? success
+        ``` text title="Example output"
+        NAME              ACTIVE BUNDLE   INSTALL STATE   AGE
+        argocd-operator                                   111s
+        ```
+
+* Describe the installed extension:
+
+    ``` terminal
+    $ kubectl describe clusterextensions
+    ```
+
+    ??? success
+        ``` text title="Example output"
+        Name:         argocd-operator
+        Namespace:
+        Labels:       <none>
+        Annotations:  <none>
+        API Version:  olm.operatorframework.io/v1alpha1
+        Kind:         ClusterExtension
+        Metadata:
+          Creation Timestamp:  2024-03-14T19:42:43Z
+          Generation:          2
+          Resource Version:    371915
+          UID:                 6f37c260-327f-4aa3-9ba1-fa1d9bc20621
+        Spec:
+          Package Name:               argocd-operator
+          Upgrade Constraint Policy:  Enforce
+        Status:
+          Conditions:
+            Last Transition Time:    2024-03-14T19:42:47Z
+            Message:                 bundledeployment status is unknown
+            Observed Generation:     2
+            Reason:                  InstallationStatusUnknown
+            Status:                  Unknown
+            Type:                    Installed
+            Last Transition Time:    2024-03-14T19:49:52Z
+            Message:                 resolved to "quay.io/operatorhubio/argocd-operator@sha256:046a9764dadcbef0b9ce67e367393fb1c8e3b1d24e361341f33ac5fb93cf32a1"
+            Observed Generation:     2
+            Reason:                  Success
+            Status:                  True
+            Type:                    Resolved
+          Resolved Bundle Resource:  quay.io/operatorhubio/argocd-operator@sha256:046a9764dadcbef0b9ce67e367393fb1c8e3b1d24e361341f33ac5fb93cf32a1
```

Comment:
Nit: Could we get an example that shows a successful installation so there is a nice example in our docs as to what the status of a "healthy" and "successful" ClusterExtension looks like?

Reply:
:facepalm: Excellent point.